### PR TITLE
Translate job offer to Spanish

### PR DESCRIPTION
**Pull request by sauloperez**
## Description

I translated the job offer to Spanish but I also took the chance to clean things up. When we first added the jobs page, we created a new layout to have separate styling. This made it possible to publish it pretty quickly but now it's time to pay the tech debt. Things are getting cumbersome.

We got one new page `/es/python_backend.html`.

## Screenshots

screenshot from 2019-01-02 16-24-31
screenshot from 2019-01-02 16-24-17

### CODE REVIEW

File: _includes/navbar_brand.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<a class='navbar-brand' href="/">
+   <img width='300px' src='{{ "/assets/img/logo.svg" | prepend:site.baseurl }}')} />
```
Comment (naoisegolden) [inline _includes/navbar_brand.html]:
```suggestion
   <img width='300' src='{{ "/assets/img/logo.svg" | prepend:site.baseurl }}')} />
```

`img` `width` and `height` attributes are ints, the size in pixels  but without the `px`.

**Review by naoisegolden — APPROVED** [review]
LGTM, only 1 correction

File: _layouts/default.html [hunk 2]
```diff
@@ -3,9 +3,36 @@
...
   {% include head.html %}
   <body>
-    {%- include header.html scrolly_nav=_posts -%}
-    {{ content }}
-    {% include footer.html %}
+    <nav class='nav navbar navbar-expand flex-row-reverse'>
+      {% include language_switcher.html %}
+
+      <ul class="navbar-nav ml-auto">
+        <li class="nav-item"><a href='/blog.html' class='nav-link'>Blog</a></li>
```
Comment (naoisegolden) [inline _layouts/default.html]: @sauloperez as far as I can see, there is no link to the jobs page. Is this by design or is it a mistake?

```suggestion
        <li class="nav-item"><a href='/blog.html' class='nav-link'>{% t pages.navigation.blog %}</a></li>
        <li class="nav-item"><a href='/jobs.html' class='nav-link'>{% t pages.navigation.jobs %}</a></li>
```

(If you accept this suggestion you'll still need to update the i18n files)

Comment (sauloperez) [reply _layouts/default.html]: Yes, still not there. That comes with the next commit. So far, only refactoring.

File: _i18n/es/python_backend.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+## Acerca de Coopdevs
+
+Coopdevs es una entidad que aporta software libre a la economía social i solidaria. Surgida desde profesionales del sector tecnológico de Barcelona con inquietudes sociales, proporciona amplios conocimientos técnicos i metodologías de trabajo en este ámbito económico. Entre muchos otros proyectos, impulsamos [Katuma](http://katuma.org/), formamos parte del equipo de desarrollo de [Open Food Network](https://openfoodnetwork.org/) y desarrollamos [TimeOverflow](https://www.timeoverflow.org/).
```
Comment (alvaro-chalar) [inline _i18n/es/python_backend.md]: `social y solidaria`, `conocimientos técnicos y metodologías`

File: _i18n/es/python_backend.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Acerca del trabajo
+
+Ahora buscamos gente con espíritu cooperativo que tenga ganas de compartir este reto aportando su experiencia de desarrollo de software en un momento importante para hacer crecer el proyecto.
+
+La candidata se unirá principalmente al equipo que trabaja en la linea de negocio de ERP para cooperativas que se desarrolla en Python. Esta unidad se encarga de acompañar y ayudar a cooperativas y otras entidades a la sistematización de sus procesos internos y su optimización con software ERP. Las tareas són:
```
Comment (alvaro-chalar) [inline _i18n/es/python_backend.md]: `línea`.....`las tareas son:`

File: _i18n/es/python_backend.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+* Saber trabajar en equipo
+* Capacidad de comunicación
+
+Algunas de las tecnologías que se usan son: Postgresql, Python, Rails, Ansible, Odoo y Git.
+
+Aspectos que valoramos muy positivamente:
+
+* Haber contribuido en proyectos open source
+* Interés en mejorar el ecosistema de la economía social y solidaria y la participación en proyectos de este ámbito
+* Experiencia con algun ERP escrito en Python
```
Comment (alvaro-chalar) [inline _i18n/es/python_backend.md]: `algún`

File: _i18n/es/python_backend.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+* Conocimientos de optimización de bases de datos relacionales
+* Conocimientos de frontend web: JavaScript, CSS y HTML
+* Familiaridad con la línea de comandos
+* Interés en Data-Drive: basar las decisiones en datos empíricos a partir de sistemas de business intelligence, analíticas y monitorización
+
+<br/>
```
Comment (alvaro-chalar) [inline _i18n/es/python_backend.md:41]: do you need this?

File: _i18n/es/python_backend.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+Aplica para esta oferta enviandonos un mail a <a href="mailto: info@coopdevs.org">info@coopdevs.org</a>
```
Comment (alvaro-chalar) [inline _i18n/es/python_backend.md]: `enviándonos`

File: _i18n/es/python_backend.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+### Únete y impulsemos la economía social y solidaria juntos!
```
Comment (alvaro-chalar) [inline _i18n/es/python_backend.md]: `Únete e impulsemos`

**Review by alvaro-chalar — CHANGES_REQUESTED** [review]

**Review by alvaro-chalar — APPROVED** [review]
:clap:

File: _i18n/es/python_backend.md (hunk 6 above)
Comment (sauloperez) [reply _i18n/es/python_backend.md:41]: I use it to separate this last sentence from the main text

**Review by sseerrggii — APPROVED** [review]

**Pull request merged**